main.js

```diff
@@ -795,6 +795,14 @@ function newGen(model) {
             },
             native: {}
         });
+        adapter.extendObject('control.fan_power', {
+            common: {
+                max: 105,
+                states: {
+                  105: "MOP"
+                },
```

Comment:
The Comma needs to be removed.
```suggestion
                }
```

Reply:
Done